lib/index.js

```diff
@@ -1,37 +1,35 @@
-'use strict';
+/**
+ * @file react-anything-sortable
+ * @author jasonslyvia
+ */
 
-var _interopRequireWildcard = function (obj) { return obj && obj.__esModule ? obj : { 'default': obj }; };
+/*eslint new-cap:0, consistent-return: 0 */
+
+'use strict';
 
 Object.defineProperty(exports, '__esModule', {
```

Comment:
In order to make `react-anything-sortable` work in IE 8, you can't use `Object.defineProperty` for non-DOM elements. Maybe you should take a look of `Babel`'s `loose mode` configuration for better compiling options.